classes/container.php

```diff
@@ -28,7 +28,12 @@ public function build(atoum\scripts\runner $script)
 
         $loader = new YamlFileLoader($this->container, new FileLocator(array(__DIR__ . '/../resources', getcwd())));
         $loader->load('services.yml');
-        $loader->load('.atoum.yml');
+
+        try {
+            $loader->load('.atoum.yml');
```

Comment:
I'd rather do a `file_exists` here and use the `try...catch` to catch Yaml parser errors.